sos/plugins/cs.py

```diff
@@ -60,69 +60,166 @@ def checkversion(self):
-                "/var/log/dirsrv/slapd-*/access",
-                "/var/log/dirsrv/slapd-*/errors"
-            ])
+                "/var/lib/rhpki-*/conf/*cfg*",
+                "/var/lib/rhpki-*/conf/*.ldif",
+            ], sizelimit=self.limit)
+
         if csversion == 8:
+            # Files containing sensitive information.
+            self.add_forbidden_path("/etc/pki-*/password.conf")
+            self.add_forbidden_path("/var/lib/pki-*/alias/key3.db")
+
+            # Get certificates from CA.
+            self.add_cmd_output([
+                "certutil -L -d /var/lib/pki-*/alias"
+            ])
+
+            # Grab all logs.
+            if self.get_option("all_logs"):
+                self.add_copy_spec([
+                    "/var/log/pki-*/",
+                    "/var/log/pki-*-install.log"
+                ])
+
+            # Grab logs for each subsystem.
             self.add_copy_spec([
-                "/etc/pki-*/CS.cfg",
-                "/var/lib/pki-*/conf/*cfg*",
                 "/var/log/pki-*/debug",
                 "/var/log/pki-*/catalina.*",
                 "/var/log/pki-*/ra-debug.log",
                 "/var/log/pki-*/transactions",
                 "/var/log/pki-*/system"
-            ])
+            ], sizelimit=self.limit)
+
+            # Grab configs for each subsystem.
+            self.add_copy_spec([
+                "/etc/pki-*/",
+                "/var/lib/pki-*/",
+            ], sizelimit=self.limit)
+
         if csversion == 9:
-            # Get logs and configs for each subsystem if installed
-            for subsystem in ('ca', 'kra', 'ocsp', 'tks', 'tps'):
+            for dirs in os.listdir("/var/lib/pki"):
+                # Files containing sensitive information.
+                self.add_forbidden_path("/etc/pki/%s/password.conf" % dirs)
+                self.add_forbidden_path("/etc/pki/%s/alias/key3.db" % dirs)
+
+                # Get certificates from CA.
+                self.add_cmd_output([
+                    "certutil -L -d /etc/pki/%s/alias" % dirs
+                ])
+
+                # Grab all logs.
+                if self.get_option("all_logs"):
+                    self.add_copy_spec([
+                        "/var/log/pki/%s" % dirs,
+                        "/var/log/pki/pki-*-spawn.*"
+                    ])
+
+                # Grab logs for each subsystem.
+                for subsystem in ("ca", "kra", "ocsp", "tks", "tps"):
+                    self.add_copy_spec([
+                        "/var/log/pki/" + dirs + "/catalina.*",
+                        "/var/log/pki/" + dirs + "/host-manager.*",
+                        "/var/log/pki/" + dirs + "/localhost.*",
+                        "/var/log/pki/" + dirs + "/manager.*",
+                        "/var/log/pki/" + dirs + "/" + subsystem + "/debug",
+                        "/var/log/pki/" + dirs + "/"
+                                        + subsystem + "/selftests",
```

Comment:
Put the plus sign at the end of previous line, like:

                        "/var/log/pki/" + dirs + "/" +
                                          subsystem + "/selftests",

to prevent the "W503 line break before binary operator" pep error.